assets/src/containers/SideDrawer.js

```diff
@@ -49,10 +49,14 @@ function SideDrawer (props) {
             to={props.path}
             key={key}
             className={classes.sideDrawerLinks}
-            selected={selectedIndex === key}
             onClick={() => setSelectedIndex(key)}
           >
-            <ListItemIcon><props.icon /></ListItemIcon>
+            <ListItemIcon>
+              {selectedIndex === key
+                ? <props.icon color='secondary' />
+                : <props.icon style={{color: '#E1E1E1'}}/>
```

Comment:
Maybe we can use the `negative` color from the palette here instead? That would keep it consistent with the rest of the application.

I would also probably format it like this, but that's a minor issue:
```
              {
                selectedIndex === key
                  ? <props.icon color='secondary' />
                  : <props.icon color='negative' />
              }
```

Reply:
I tried using the negative color but thought that there wasn't enough contrast between secondary and negative colors. I attached a screenshot of the SideBar using the secondary and negative as the colors. Let me know if you think that there is enough contrast and that I should change it back. Thanks.

<img width="267" alt="Screen Shot 2020-03-13 at 10 21 03 AM" src="https://user-images.githubusercontent.com/33735083/76629331-7c74c480-6514-11ea-94e0-66f648fe9c45.png">

Reply:
@jennlove-um, do you think the contrast is worse here between `secondary` (the blue) and `negative` (the gray). I kind of see what @vjcao is saying, but I'd rather stick to the same colors throughout if we can.